.gitignore

```diff
@@ -1,3 +1,4 @@
+MASTER
```

Comment:
Hi @oliviagallucci 

I think you have a spurious add here? Probably want to

1. remove that line
2. commit
3. rebase all 3 commits, squashing / fixup so it's a single commit
4. force push

When you force push to your fork's branch, it'll auto-update this PR

Hope it helps!

Reply:
Yes! Will look into it.